_posts/2016-04-27-discounted-goruco-tickets-for-underrepresented-people.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+excerpt_separator: <!--more-->
+---
+
+<p>
+  This year we are offering discounted tickets for those who self-identify
+  as underrepresented in technology via a discount code.</p>
+
+<h3>Why offer a discount at all?</h3>
+
+<p>
+  GORUCO's organizing committee believes our event
+  should reflect and model the values of our communities.
+  Diversity and inclusivity have rightly risen to prominence
+  as major points of concern in both our local community and
+  the broader Ruby community.</p>
+
+<!--more-->
+
+<h3>How to?</h3>
+
+<p>
+  The discount code is: "diversity."
+  You can enter it on this page as you buy your ticket.
+  It entitles the purchaser to a $100 discount in ticket price.</p>
+
+<p>
+  The choice of a discount code reflects the nature of this year's experiment
+  to sell tickets to the conference in three weekly batches.
+  In past years, tickets have sold out so quickly
+  that some have missed the opportunity to buy.
+  This seems likely to particularly affect
+  those who are not immediately certain they wish to attend.</p>
```

Comment:
Suggested change: _This particularly impacted those people who were not immediately sure that they were able to attend._